<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>672</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QFrame" name="ConnectionFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>400</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>500</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label_58">
             <property name="text">
              <string>Enter Connection Details below</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTabWidget" name="ConnectionTypeTabs">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="tabPosition">
              <enum>QTabWidget::North</enum>
             </property>
             <property name="tabShape">
              <enum>QTabWidget::Rounded</enum>
             </property>
             <property name="currentIndex">
              <number>1</number>
             </property>
             <property name="tabsClosable">
              <bool>false</bool>
             </property>
             <property name="tabBarAutoHide">
              <bool>false</bool>
             </property>
             <widget class="QWidget" name="tcp_4">
              <attribute name="title">
               <string>TCP/IP</string>
              </attribute>
              <widget class="QWidget" name="formLayoutWidget_7">
               <property name="geometry">
                <rect>
                 <x>30</x>
                 <y>10</y>
                 <width>291</width>
                 <height>321</height>
                </rect>
               </property>
               <layout class="QFormLayout" name="FormLayout_8">
                <item row="0" column="1">
                 <widget class="QLineEdit" name="tcp_name">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_59">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>Name</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="tcp_host">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_60">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>Host</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="tcp_username">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_61">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>Username</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="tcp_password">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="echoMode">
                   <enum>QLineEdit::Password</enum>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_62">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>Password</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QLineEdit" name="tcp_database">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="placeholderText">
                   <string>Optional</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_63">
                  <property name="text">
                   <string>Database</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="QLineEdit" name="tcp_port">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="placeholderText">
                   <string>3306</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="label_64">
                  <property name="text">
                   <string>Port</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
             <widget class="QWidget" name="ssh_4">
              <attribute name="title">
               <string>SSH</string>
              </attribute>
              <widget class="QWidget" name="formLayoutWidget_8">
               <property name="geometry">
                <rect>
                 <x>30</x>
                 <y>10</y>
                 <width>291</width>
                 <height>341</height>
                </rect>
               </property>
               <layout class="QFormLayout" name="FormLayout_9">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_65">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>Name</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="ssh_name">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_66">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>Host</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="ssh_host">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_67">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>Username</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="ssh_username">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_68">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string>Password</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="ssh_password">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="echoMode">
                   <enum>QLineEdit::Password</enum>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_69">
                  <property name="text">
                   <string>Database</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QLineEdit" name="ssh_database">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="placeholderText">
                   <string>Optional</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="label_70">
                  <property name="text">
                   <string>Port</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="QLineEdit" name="ssh_port">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="placeholderText">
                   <string>3306</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="1">
                 <spacer name="verticalSpacer_5">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="8" column="0">
                 <widget class="QLabel" name="label_71">
                  <property name="text">
                   <string>SSH Host</string>
                  </property>
                 </widget>
                </item>
                <item row="8" column="1">
                 <widget class="QLineEdit" name="ssh_ssh_host">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="echoMode">
                   <enum>QLineEdit::Normal</enum>
                  </property>
                 </widget>
                </item>
                <item row="9" column="0">
                 <widget class="QLabel" name="label_72">
                  <property name="text">
                   <string>SSH User</string>
                  </property>
                 </widget>
                </item>
                <item row="9" column="1">
                 <widget class="QLineEdit" name="ssh_ssh_user">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="echoMode">
                   <enum>QLineEdit::Normal</enum>
                  </property>
                 </widget>
                </item>
                <item row="10" column="0">
                 <widget class="QLabel" name="label_73">
                  <property name="text">
                   <string>SSH Password</string>
                  </property>
                 </widget>
                </item>
                <item row="10" column="1">
                 <widget class="QLineEdit" name="ssh_ssh_password">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="echoMode">
                   <enum>QLineEdit::Password</enum>
                  </property>
                 </widget>
                </item>
                <item row="11" column="0">
                 <widget class="QLabel" name="label_74">
                  <property name="text">
                   <string>SSH Port</string>
                  </property>
                 </widget>
                </item>
                <item row="11" column="1">
                 <widget class="QLineEdit" name="ssh_ssh_port">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="echoMode">
                   <enum>QLineEdit::Normal</enum>
                  </property>
                  <property name="placeholderText">
                   <string>Optional</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QPushButton" name="ConnectionSaveButton">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ConnectionTestButton">
             <property name="text">
              <string>Test Connection</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ConnectionButton">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="SavedConnections" stretch="0,0">
     <property name="spacing">
      <number>-1</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="NewConnectionButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>New Connection</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignTop">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Favorites</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
